src/js/index.js

```diff
@@ -1,5 +1,167 @@
-(function () {
+/*global $ */
+'use strict';
 
-  console.log('WOLOX');
+//Manejo de animaciones cuando abre el sitio
+function openBars(time){
+	$('.copyright').animate({
```

Comment:
another way of doing this is by adding classes in js that generate a css transition, this way you can separate style from functionallity in a better way.